src/runners/bato/parser.ts

```diff
@@ -255,14 +255,7 @@ export class Parser {
     const imgAccListString = AES.decrypt(batoWord, evaluatedPass).toString(
       enc.Utf8
     );
-
-    if (!imgAccListString) throw new Error("Invalid Acc List String");
-
     const imgAccList: string[] = JSON.parse(imgAccListString);
```

Comment:
what happens if you parse a empty object or string?

Reply:
On my end this is always `[ ]`. Just an empty array, JSON.parse _really_ doesn't care much about what you give it.